Justfile

```diff
@@ -30,3 +34,9 @@ compile_notebooks mode="release":
 
 release:
   python setup.py sdist
+
+type-check path=".":
+  python .devtools/type-check {{ invocation_directory() + "/" + path }}
+
+license-check path=".":
+  python .devtools/license check {{ invocation_directory() + "/" + path }}
```

Comment:
no `just style-check`? :-)

Reply:
Yes, why not.

Could also have `just check`, which checks everything.